docker-images/cts-base/Dockerfile1

```diff
@@ -6,7 +6,6 @@ ARG ANT_VERSION=1.10.5
 ARG MAVEN_BASE_URL=https://apache.osuosl.org/maven/maven-3/${MAVEN_VERSION}/binaries
 ARG ANT_BASE_URL=https://apache.osuosl.org/ant/binaries/
 
-ADD jwsdp-1.3.tar /opt
```

Comment:
Dockerfile1 is a historic file and refers to the previous version of docker image deployed. We are currently in Dockerfile2 which does not have jwsdp-1.3.tar, hence can keep it.

Reply:
done.